### PR TITLE
Update config.json

### PR DESCRIPTION
**Pull request by tim-hub**
Add a simple optional list, mainly based on this research:

[Android OS Privacy Under the Loupe – A Tale from the East](https://arxiv.org/pdf/2302.01890.pdf)

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1846,6 +1846,17 @@
...
+    },
+    {
+      "vname": "Tiny CN Block List Replenish",
+      "group": "Security",
+      "format": "domains",
+      "url": [
+        "https://raw.githubusercontent.com/tim-hub/tiny-cn-blocklist-replenish/master/common.txt",
+        "https://raw.githubusercontent.com/tim-hub/tiny-cn-blocklist-replenish/master/others.txt"
+      ],
+      "pack": ["privacy", "gambling"],
```
Comment (ignoramous) [inline config.json:1858]: Thanks. Label:

"liteprivacy" if it isn't an aggressive list (shouldn't break any major functionality of popular apps)

"aggressiveprivacy" if it breaks one or major two things here and there 

"extremeprivacy" if none of the above.

Comment (tim-hub) [inline config.json:1858]:
```suggestion
      "pack": ["liteprivacy", "gambling"],
```

File: config.json [hunk 2]
```diff
@@ -1846,6 +1846,17 @@
       "url": "https://github.com/cbuijs/accomplist/raw/master/tlds/plain.black.domain.list",
       "pack": ["spam"],
       "level": [2]
+    },
+    {
+      "vname": "Tiny CN Block List Replenish",
+      "group": "Security",
```
Comment (ignoramous) [inline config.json:1852]: This should be "Privacy"?

Comment (tim-hub) [reply config.json:1852]: Yep, will update it

Comment (tim-hub) [inline config.json:1852]:
```suggestion
      "group": "Privacy",
```

**Comment by ignoramous** [comment]
Thanks

**Pull request merged**